docs/src/components/sidenav.js

```diff
@@ -1,18 +1,39 @@
 import React from 'react'
 import { StaticQuery, graphql } from 'gatsby'
-import styled from 'styled-components'
+import styled, { css } from 'styled-components'
 
 import Link from './link'
 
-const NavWrapper = styled.nav`
-`
+const NavWrapper = styled.nav``
 
 const NavList = styled.ul`
   list-style: none;
   margin: 0;
+  padding: 0;
+`
+
+const NavItem = styled.li`
+  padding-left: 1em;
+  & a {
+    text-decoration: none;
+  }
+
+  ${props =>
+    props.active
+      ? css`
+          border-left: 1px solid ${props.theme.highlight};
+          & a {
+            text-decoration: underline;
+          }
+        `
+      : css``};
 `
 
-const NavItem = styled.li``
+const SubNav = styled.ul`
+  list-style: none;
+  margin: 0;
+  padding: 0;
+`
 
 const Nav = props => (
   <NavWrapper {...props}>
```

Comment:
Out of curiosity, what's the purpose of passing `props` into `NavWrapper`?

Reply:
@dagoof ?